js/listNumbers.js

```diff
@@ -1,33 +1,32 @@
-var decimalBullet = Array.apply(null, Array(100)).map(function (a, i) {
-  return '' + (i + 1);
-});
-var alphaBullet = Array.apply(null, Array(26)).map(function (a, i) {
-  return String.fromCharCode('a'.charCodeAt(0) + i);
-});
+'use strict';
+let decimalBullet = Array.apply(null, Array(100)).map((a, i) => '' + (i + 1));
+let alphaBullet = Array.apply(null, Array(26)).map((a, i) =>
+  String.fromCharCode('a'.charCodeAt(0) + i)
+);
 
 // prettier-ignore
-var romanBullet = ['i', 'ii', 'iii', 'iv', 'v', 'vi', 'vii', 'viii', 'ix', 'x', 'xi', 'xii', 'xiii', 'xiv', 'xv', 'xvi', 'xvii', 'xviii', 'xix', 'xx', 'xxi', 'xxii', 'xxiii', 'xxiv', 'xxv'];
+let romanBullet = ['i', 'ii', 'iii', 'iv', 'v', 'vi', 'vii', 'viii', 'ix', 'x', 'xi', 'xii', 'xiii', 'xiv', 'xv', 'xvi', 'xvii', 'xviii', 'xix', 'xx', 'xxi', 'xxii', 'xxiii', 'xxiv', 'xxv'];
 // prettier-ignore
-var bullets = [decimalBullet, alphaBullet, romanBullet, decimalBullet, alphaBullet, romanBullet];
+let bullets = [decimalBullet, alphaBullet, romanBullet, decimalBullet, alphaBullet, romanBullet];
 
 function addStepNumberText(ol, parentIndex) {
-  for (var i = 0; i < ol.children.length; ++i) {
-    var child = ol.children[i];
-    var index = parentIndex.concat([i]);
-    var applicable = bullets[Math.min(index.length - 1, 5)];
-    var span = document.createElement('span');
+  for (let i = 0; i < ol.children.length; ++i) {
+    let child = ol.children[i];
+    let index = parentIndex.concat([i]);
+    let applicable = bullets[Math.min(index.length - 1, 5)];
+    let span = document.createElement('span');
```

Comment:
none of these are reassigned; can they all be const? (`prefer-const` will autofix these, if that's the style we want)
```suggestion
  for (let i = 0; i < ol.children.length; ++i) {
    const child = ol.children[i];
    const index = parentIndex.concat([i]);
    const applicable = bullets[Math.min(index.length - 1, 5)];
    const span = document.createElement('span');
```

Reply:
Ehhhh I am not really a fan of prefer-const.